pyproject.toml

```
@@ -238,3 +238,27 @@ warn_unreachable = false
 warn_redundant_casts = true
 warn_unused_ignores = true
 warn_unused_configs = true
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tool.pixi.pypi-dependencies]
+numcodecs = { path = ".", editable = true }
+
+[tool.pixi.environments]
+default = { solve-group = "default" }
+crc32c = { features = ["crc32c"], solve-group = "default" }
+docs = { features = ["docs"], solve-group = "default" }
+msgpack = { features = ["msgpack"], solve-group = "default" }
+pcodec = { features = ["pcodec"], solve-group = "default" }
+test = { features = ["test"], solve-group = "default" }
```

```suggestion
test = { features = ["test", "crc32c", "msgpack", "pcodec", "zfpy"], solve-group = "default" }
```

Add optional dependencies for the test environment.